site/docs/pages/wallet/types.mdx

```diff
@@ -86,3 +86,18 @@ export type WalletDropdownLinkReact = {
   target?: string;
 };
 ```
+
+## `WalletDropdownFundLinkReact`
+
+```ts
+export type WalletDropdownFundLinkReact = {
+  className?: string; // Optional className override for the element
+  href: string;
+  icon?: ReactNode;
+  rel?: string;
+  target?: string;
+  text?: string;
+  type?: 'popup' | 'link';
```

Comment:
Let's add comments to each of one of them. As I don't quite follow what's going on with some of those properties.